components/table/hooks/useFilter/FilterDropdown.tsx

```diff
@@ -92,20 +93,22 @@ function FilterDropdown<RecordType>(props: FilterDropdownProps<RecordType>) {
     filterState && filterState.filteredKeys,
   );
   const [filteredKeys, setFilteredKeys] = React.useState(propFilteredKeys || []);
+  const [getFilteredKeysSync, setFilteredKeysSync] = useSyncState(propFilteredKeys || []);
```

Comment:
`const [filteredKeys, setFilteredKeys] = React.useState(propFilteredKeys || []);` can be removed now.